scripts/pi-hole/js/queries.js

```diff
@@ -293,18 +293,32 @@ $(document).ready(function() {
                 var content = $("td:eq(5)", row).html();
                 $("td:eq(5)", row).html(content + " (" + (0.1*data[7]).toFixed(1)+"ms)");
             }
-
+            var timestamp = Math.floor(parseFloat(data[0])/1e6);
+            $("td:eq(0)", row).html(moment.unix(timestamp).format("Y-MM-DD [<br class='hidden-lg'>]HH:mm:ss z"));
         },
         dom: "<'row'<'col-sm-12'f>>" +
              "<'row'<'col-sm-4'l><'col-sm-8'p>>" +
              "<'row'<'col-sm-12'<'table-responsive'tr>>>" +
              "<'row'<'col-sm-5'i><'col-sm-7'p>>",
-        "ajax": {"url": APIstring, "error": handleAjaxError },
+        "ajax": {
+            "url": APIstring,
+            "error": handleAjaxError,
+            "dataSrc": function(data){
+                var new_data = new Array();
+                for(obj of data.data)
+                {
+                    obj[0] *= parseInt(1e6);
+                    obj[0] += dataIndex++;
+                    new_data.push(obj);
+                }
+                return new_data;
+            }
+        },
         "autoWidth" : false,
         "processing": true,
         "order" : [[0, "desc"]],
         "columns": [
-            { "width" : "15%", "render": function (data, type, full, meta) { if(type === "display"){return moment.unix(data).format("Y-MM-DD [<br class='hidden-lg'>]HH:mm:ss z");}else{return data;} }},
```

Comment:
Could this render function have instead been modified? When the type is "display", do the division and display the time. Otherwise (for sorting I assume) return the data.